frontend_tests/node_tests/settings_bots.js

```diff
@@ -21,9 +21,7 @@ const bot_data_params = {
 
 const avatar = mock_esm("../../static/js/avatar");
 
-function ClipboardJS(sel) {
-    assert.equal(sel, "#copy_zuliprc");
-}
+function ClipboardJS() {}
```

Comment:
What this does is effectively just not test the code since there's no assertion anymore. Now, of course, that assertion no longer works since we've changed the code, but try and look at how `./frontend_tests/node_tests/rendered_markdown.js` mocks clipboard and see if we can actually assert.equal with the `sel` we're getting.

Reply:
@YashRE42 thanks for the feedback. This is what I had in mind:
`function ClipboardJS(sel) {
    const expected_values = ["#copy_zuliprc", "#copy_bot_email", "#copy_api_key"];
    assert(expected_values.includes(sel));
}`

The node tests are passing on my machine with this.

Reply:
I don't think that's the right testing strategy. Can you tell how it's different from `./frontend_tests/node_tests/rendered_markdown.js`?

Reply:
Are you referring to the `assert_clipboard_setup()` function on line 384 in `./frontend_tests/node_tests/rendered_markdown.js`?

Reply:
Do I have to use `assert.equal()` instead of `assert()` for consistency?

Reply:
I got an email about a comment saying:
> Are you referring to the `assert_clipboard_setup()` function on line 384 in `./frontend_tests/node_tests/rendered_markdown.js`?

but I don't see it on this thread anymore.... but that was exactly what I had in mind. Each case should define what it expects and assert that what it's getting is what it expects, just like the cases in `rendered_markdown` do.

Reply:
I thought I was going in the wrong direction so I deleted the comment. 
I can't figure out what the `to_$` and `siblings` properties are doing. Do I have to use this syntax or should I think of a simpler approach?

Reply:
Okay, I'll start a CZO topic, mentioning Steve and figure out a better testing strategy (possibly for both files).

Until that's figured out I think it's fine for us to regress with:
```diff
-function ClipboardJS(sel) {
-    assert.equal(sel, "#copy_zuliprc");
-}
+function ClipboardJS() {}
```

So, you can just address https://github.com/zulip/zulip/pull/20733#issuecomment-1011477707.